monai/networks/nets/hovernet.py

```diff
@@ -387,33 +386,9 @@ class HoVerNet(nn.Module):
         dropout_prob: dropout rate after each dense layer.
     """
 
```

Comment:
I'm wondering if you define
```py
Mode = HoVerNetMode
Branch = HoVerNetBranch
```
would make it nonbreaking and avoid any changes in the unit tests.